cmd/mount.go

```diff
@@ -287,6 +288,7 @@ func useConfiguration() {
 
 	svfs.SwiftConnection.AuthToken = viper.GetString("os_auth_token")
 	svfs.SwiftConnection.StorageUrl = viper.GetString("os_storage_url")
+	svfs.SwiftConnection.Internal = viper.GetBool("os_internal_endpoint")
```

Comment:
Same thing here, remove unnecessary viper call since the option should only handled by cobra.